diff --git a/src/crypto/src/kestrel_wasm.js b/src/crypto/src/kestrel_wasm.js
index 670dc65..c1c4870 100644
--- a/src/crypto/src/kestrel_wasm.js
+++ b/src/crypto/src/kestrel_wasm.js
@@ -497,7 +497,7 @@ async function __wbg_init(input) {
     if (wasm !== undefined) return wasm;
 
     if (typeof input === 'undefined') {
-        input = new URL('kestrel_wasm_bg.wasm', import.meta.url);
+        throw new Error("wasm input data not defined.");
     }
     const imports = __wbg_get_imports();
 
